dev/tasks/README.md

```diff
@@ -60,6 +60,9 @@ submission. The tasks are defined in `tasks.yml`
    repository](https://help.github.com/articles/creating-a-new-repository)
 2. Enable [TravisCI](https://travis-ci.org/getting_started) and
    [Appveyor](https://www.appveyor.com/docs/) integrations on it
+
+   - turn off Travis' [auto cancellation](https://docs.travis-ci.com/user/customizing-the-build/#Building-only-the-latest-commit) feature on branches:
```

Comment:
Is the colon at the end supposed to introduce something?

Reply:
Not of course.